src/data/html-quiz.ts

```diff
@@ -9,7 +21,7 @@ const htmlQuiz = [
     Explanation:
       "You can use the <script> element to link to an external 3rd party JavaScript library and connect to its API functionality.",
     Link:
-      "https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Third_party_APIs"
+      "https://developer.mozilla.org/en-US, /docs/Learn/JavaScript/Client-side_web_APIs/Third_party_APIs"
```

Comment:
This change creates a broken link.
Please change it back to the way it was 👍

Reply:
This link is still incorrect so I am unresolving this conversation. 
Once it is changed back to the way it was, then this PR can be reviewed again 👍